base/cron.py

```diff
@@ -14,3 +14,15 @@ class CronCancellaFileScaduti(CronJobBase):
     def do(self):
         n = Allegato.pulisci()
         print("Sono stati rimossi %d file scaduti." % n)
+
+
+class CronApprovaNegaAuto(CronJobBase):
```

Comment:
E' necessario aggiungere la classe in `jorvik/settings.py` in `CRON_CLASSES` per abilitare l'esecuzione del cron.